### PR TITLE
feat(config): add DJANGO_VITE_STATIC_URL_PREFIX

### LINKED ISSUE #21: Proposal: Add DJANGO_VITE_ASSETS_SUBDIRECTORY
Hi !

## Current issue

I would like to build my vite assets inside a static's subdirectory.
For example with (`build.outDir: '../static/build/'`) : 
- `app/static/build/main.js`
- `app/static/build/_vendor.c8732.js`
etc...

This architecture would avoid conflicts when sharing static directory with other static files.

It does not seems possible at the moment because assets are inserted from the STATIC_URL only: https://github.com/MrBin99/django-vite/blob/116f61f17ce4622d8e2f83c0bae9a8dbf014a31e/django_vite/templatetags/django_vite.py

## Implementation

Add a new configuration in django settings.py : `DJANGO_VITE_ASSETS_SUBDIRECTORY`.

e.g.
```python
DJANGO_VITE_ASSETS_SUBDIRECTORY = "build"
```

If this variable is defined, assets path would be defined as 
```python
urljoin(
    settings.STATIC_URL,
    settings.DJANGO_VITE_ASSETS_SUBDIRECTORY,
    manifest_entry["file"]
)
````

Would you approve such a PR? I can implement it.
If not, would you have an alternative solution for such a static files architecture?

Thanks!

### PR DESCRIPTION
**Pull request by gnuletik**
Fixes #21

### CODE REVIEW

File: django_vite/templatetags/django_vite.py [hunk 1]
```diff
@@ -53,6 +60,10 @@
     settings, "DJANGO_VITE_LEGACY_POLYFILLS_MOTIF", "legacy-polyfills"
 )
 
+DJANGO_VITE_STATIC_URL = path_join(
+    settings.STATIC_URL, DJANGO_VITE_STATIC_URL_PREFIX, ""
```
Comment (MrBin99) [inline django_vite/templatetags/django_vite.py:64]: I don't understand the third parameter (empty string) here.

File: README.md [hunk 2]
```diff
@@ -175,6 +175,23 @@ You can redefine those variables in your `settings.py` :
   then it is in your `STATIC_ROOT` after you collected your
   [static files](https://docs.djangoproject.com/en/3.1/howto/static-files/).
 - `DJANGO_VITE_LEGACY_POLYFILLS_MOTIF` : The motif used to find the assets for polyfills inside the `manifest.json` (only if you use [@vitejs/plugin-legacy](https://github.com/vitejs/vite/tree/main/packages/plugin-legacy)).
+- `DJANGO_VITE_STATIC_URL_PREFIX` : prefix directory of your static files built by Vite.
```
Comment (MrBin99) [inline README.md:178]: Can you provide the default value for this variable in the doc.
Can you make sub-bullet points for each item "You ...", sorry I'm a little maniac 😊

File: django_vite/templatetags/django_vite.py (hunk 1 above)
Comment (gnuletik) [reply django_vite/templatetags/django_vite.py:64]: It adds a leading slash to the path if the `DJANGO_VITE_STATIC_URL_PREFIX` doesn't contains one.

It is mandatory that `DJANGO_VITE_STATIC_URL` has a leading slash in order to be used with `urljoin`.
If there is no leading slash the last path would be omitted.

e.g.
```python
>>> from urllib.parse import urljoin

# bad
>>> urljoin("/static/bundler", "main.js")
'/static/main.js'

# good
>>> urljoin("/static/bundler/", "main.js")
'/static/bundler/main.js'

```

File: README.md (hunk 2 above)
Comment (gnuletik) [reply README.md]: Done! It should be better now.
I totally get it! I'm a little maniac too :)

**Comment by gnuletik** [comment]
Thanks for the fast review!

File: django_vite/templatetags/django_vite.py (hunk 1 above)
Comment (MrBin99) [reply django_vite/templatetags/django_vite.py:64]: Ok I see thanks !

File: README.md (hunk 2 above)
Comment (MrBin99) [reply README.md]: 😂

**Comment by MrBin99** [comment]
Thank you !

**Comment by gnuletik** [comment]
Thanks for the merge & release !

**Pull request merged**